pipeline/matcher/src/test/scala/uk/ac/wellcome/platform/matcher/matcher/WorkMatcherTest.scala

```diff
@@ -75,7 +75,9 @@ class WorkMatcherTest
               matcherResult shouldBe
                 MatcherResult(
                   Set(MatchedIdentifiers(Set(WorkIdentifier(workId, 1)))))
-              get[WorkNode](dynamoClient, graphTable.name)('id -> workId) shouldBe None
+              get[WorkNode](dynamoClient, graphTable.name)('id -> workId)
+                .map(_.right.get) shouldBe Some(
```

Comment:
FWIW, we've been starting to use EitherValues and `_.right.value` instead of `_.right.get` in tests while you were away; one of Robert's discoveries, I think.

I don't see a massive difference, but you might want to consider it: http://www.scalatest.org/user_guide/using_EitherValues

Reply:
Ah nice thanks